### PR TITLE
Update apipie-rails to 0.5.6

### PR DESCRIPTION
**Pull request by pcreech**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [x] 1.16
* [ ] 1.15
* [ ] 1.14

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: rubygem-apipie-rails/rubygem-apipie-rails.spec [hunk 1]
```diff
@@ -84,6 +84,9 @@ cp -a .%{gem_dir}/* \
 %{gem_instdir}/%{gem_name}.gemspec
 
 %changelog
+* Wed Dec 06 2017 Patrick Creech <pcreech@redhat.com> - 0.5.6-1
+- Update apipie-rails to 0.5.6
```
Comment (ehelms) [inline rubygem-apipie-rails/rubygem-apipie-rails.spec]: You can leave this part out, our process involves tito tagging after we merge which will generate the changelog.

Comment (pcreech) [reply rubygem-apipie-rails/rubygem-apipie-rails.spec]: Removed

**Comment by ekohl** [comment]
[test]

**Comment by dLobatog** [comment]
http://koji.katello.org/koji/taskinfo?taskID=61091

**Pull request merged**